easyDiffractionApp/Logic/Fitter.py

```diff
@@ -64,6 +71,16 @@ def _setFitResults(self, res):
             "GOF":     float(res.goodness_of_fit),
             "redchi2": float(res.reduced_chi)
         }
+        self.setFitResults()
+
+    def onFailed(self, ex):
```

Comment:
This `ex` variable is not used. Would it be more useful to at least print it out for debugging? In the future we will also need to display this error message to the user in a popup window.

Reply:
fixed